app/helpers/protocol_helper.rb

```diff
@@ -73,4 +73,26 @@ def consolidated_one_time_fee_line_items protocol
 
     line_items
   end
+
+  def current_tab? current_tab, tab_name, services_present = true
```

Comment:
It looks good, except you would need to add the study level activities default logic too:
'active' if (current_tab.present? && current_tab == tab_name) || (services_present && tab_name == 'study_schedule') || (!services_present && tab_name == 'study_level_activities')